### PR TITLE
Library documentation

### PR DESCRIPTION
**Pull request by ggajoch**

### CODE REVIEW

File: src/CanSatKitRadio.h [hunk 1]
```diff
@@ -63,21 +84,97 @@ class Radio {
...
   static bool begin();
+
+  /**
+   * @brief Disable debug messages on SerialUSB.
+   */
   static void disable_debug();
   
+  /**
+   * @brief Put frame into the transmit buffer.
+   * @return `true` if frame put into buffer. `false` if not enught space in the buffer.
```
Comment (begreeen) [inline src/CanSatKitRadio.h]: a typo: enught -> enough

File: src/CanSatKitRadio.h [hunk 2]
```diff
@@ -63,21 +84,97 @@ class Radio {
...
+   */
   static bool transmit(Frame frame);
+
+  /**
+   * @brief Put String str into the transmit buffer.
+   * @return `true` if frame put into buffer. `false` if not enught space in the buffer.
```
Comment (begreeen) [inline src/CanSatKitRadio.h]: a typo: enught -> enough

File: src/CanSatKitRadio.h [hunk 3]
```diff
@@ -63,21 +84,97 @@ class Radio {
...
+   */
   static bool transmit(String str);
+
+  /**
+   * @brief Put string str into the transmit buffer.
+   * @return `true` if frame put into buffer. `false` if not enught space in the buffer.
```
Comment (begreeen) [inline src/CanSatKitRadio.h]: a typo: enught -> enough

File: src/CanSatKitRadio.h [hunk 4]
```diff
@@ -63,21 +84,97 @@ class Radio {
...
+   */
   static bool transmit(const char* str);
+
+  /**
+   * @brief Put binary data into the transmit buffer (byte table of length length).
+   * @return `true` if frame put into buffer. `false` if not enught space in the buffer.
```
Comment (begreeen) [inline src/CanSatKitRadio.h]: a typo: enught -> enough

File: src/CanSatKitBMP280.h [hunk 5]
```diff
@@ -5,83 +5,132 @@
...
-			// 4 = Other error
-			
-		char readTemperatureAndPressure(double& T, double& P);
+		/**
+		 * @brief Set oversampling ratio.
+		 * Should be one of the following: 0, 1, 2, 3, 4, 16.
+		 * 
+		 * \rst
+		 * +---------------+-------------+
+         * | Oversampling  | Delay (ms)  |
```
Comment (begreeen) [inline src/CanSatKitBMP280.h]: Consider changing `Delay (ms)` to `Conversion time (ms)`

File: src/CanSatKitBMP280.h [hunk 6]
```diff
@@ -5,83 +5,132 @@
...
+		 * | 4             | 45          |
+         * +---------------+-------------+
+		 * | 16            | 80          |
+         * +---------------+-------------+
+		 * 
+		 * \endrst
+		 * 
+		 * Higher oversampling means longer measurement, but more accurate result
+		 * 
+		 * @param oversampling radio
```
Comment (begreeen) [inline src/CanSatKitBMP280.h]: radio?

File: src/CanSatKitBMP280.h [hunk 7]
```diff
@@ -5,83 +5,132 @@
...
+		 * @return bool Measurement status
+		 */
+		bool measureTemperatureAndPressure(double& T, double& P);
+
+
+		/**
+		 * @brief Begin a measurement cycle.
+		 * This function returns the delay before result will be available to read.
+		 * 
+		 * @return unsigned int delay to read
```
Comment (begreeen) [inline src/CanSatKitBMP280.h]: consider `unsigned int delay to read measurement result (ms)`

**Review by begreeen — CHANGES_REQUESTED** [review]

File: src/CanSatKitBMP280.h (hunk 7 above)
Comment (ggajoch) [reply src/CanSatKitBMP280.h]: done

File: src/CanSatKitBMP280.h (hunk 6 above)
Comment (ggajoch) [reply src/CanSatKitBMP280.h]: done

File: src/CanSatKitBMP280.h (hunk 5 above)
Comment (ggajoch) [reply src/CanSatKitBMP280.h]: done

File: src/CanSatKitRadio.h (hunk 4 above)
Comment (ggajoch) [reply src/CanSatKitRadio.h]: done

File: src/CanSatKitRadio.h (hunk 3 above)
Comment (ggajoch) [reply src/CanSatKitRadio.h]: done

File: src/CanSatKitRadio.h (hunk 2 above)
Comment (ggajoch) [reply src/CanSatKitRadio.h]: done

File: src/CanSatKitRadio.h (hunk 1 above)
Comment (ggajoch) [reply src/CanSatKitRadio.h]: done

**Review by begreeen — APPROVED** [review]

**Pull request merged**